src/types/gameConsole.ts

```diff
@@ -12,109 +12,116 @@ export default class GameConsole {
     // Arduboy
-    new GameConsole(/Arduboy/i, ['.arduboy', '.hex'], 'arduboy', 'Arduboy', undefined),
+    new GameConsole(/Arduboy/i, ['.arduboy', '.hex'], 'arduboy', 'Arduboy', undefined, 'arduboy'),
     // Atari
-    new GameConsole(/800|8-bit Family/, ['.atr', '.atx'], undefined, 'ATARI800', 'EIGHTHUNDRED'),
-    new GameConsole(/2600/, ['.a26', '.act', '.pb', '.tv', '.tvr', '.mn', '.cv', '.eb', '.ef', '.efr', '.ua', '.x07', '.sb'], '2600', 'Atari2600', 'ATARI'),
-    new GameConsole(/5200/, ['.a52'], undefined, 'Atari5200', 'FIFTYTWOHUNDRED'),
-    new GameConsole(/7800/, ['.a78'], '7800', 'Atari7800', 'SEVENTYEIGHTHUNDRED'),
-    new GameConsole(/Jaguar/i, ['.j64'], undefined, undefined, 'JAGUAR'),
-    new GameConsole(/Lynx/i, ['.lnx', '.lyx'], undefined, 'AtariLynx', 'LYNX'),
-    new GameConsole(/Atari (- )?ST/i, ['.msa', '.st', '.stx'], undefined, 'AtariST', 'ATARIST'),
+    new GameConsole(/800|8-bit Family/, ['.atr', '.atx'], undefined, 'ATARI800', 'EIGHTHUNDRED', 'atari800'),
+    new GameConsole(/2600/, ['.a26', '.act', '.pb', '.tv', '.tvr', '.mn', '.cv', '.eb', '.ef', '.efr', '.ua', '.x07', '.sb'], '2600', 'Atari2600', 'ATARI', 'atari2600'),
+    new GameConsole(/5200/, ['.a52'], undefined, 'Atari5200', 'FIFTYTWOHUNDRED', 'atari5200'),
+    new GameConsole(/7800/, ['.a78'], '7800', 'Atari7800', 'SEVENTYEIGHTHUNDRED', 'atari7800'),
+    new GameConsole(/Jaguar/i, ['.j64'], undefined, undefined, 'JAGUAR', 'jaguar'),
+    new GameConsole(/Lynx/i, ['.lnx', '.lyx'], undefined, 'AtariLynx', 'LYNX', 'lynx'),
+    new GameConsole(/Atari (- )?ST/i, ['.msa', '.st', '.stx'], undefined, 'AtariST', 'ATARIST', 'atarist'),
     // Bally
-    new GameConsole(/Astrocade/i, [/* '.bin' */], undefined, 'Astrocade', undefined),
+    new GameConsole(/Astrocade/i, [/* '.bin' */], undefined, 'Astrocade', undefined, 'astrocde'),
     // Bandai
-    new GameConsole(/WonderSwan/i, ['.ws'], 'wonderswan', 'WonderSwan', 'WS'),
-    new GameConsole(/WonderSwan Color/i, ['.wsc'], 'wonderswan', 'WonderSwan', 'WS'),
+    new GameConsole(/WonderSwan/i, ['.ws'], 'wonderswan', 'WonderSwan', 'WS', 'wswan'),
+    new GameConsole(/WonderSwan Color/i, ['.wsc'], 'wonderswan', 'WonderSwan', 'WS', 'wswanc'),
     // Bit Corporation
-    new GameConsole(/Gamate/i, [/* '.bin' */], 'gamate', 'Gamate', undefined),
+    new GameConsole(/Gamate/i, [/* '.bin' */], 'gamate', 'Gamate', undefined, 'gamate'),
     // Capcom
     // TODO(cemmer): CPS1, CPS2, CPS3
     // Casio
-    new GameConsole(/PV-?1000/i, [/* '.bin' */], undefined, 'Casio_PV-1000', undefined),
+    new GameConsole(/PV-?1000/i, [/* '.bin' */], undefined, 'Casio_PV-1000', undefined, 'pv1000'),
     // Commodore
-    new GameConsole(/Amiga/i, [], 'amiga', 'Amiga', 'AMIGA'),
-    new GameConsole(/Commodore 64/i, ['.crt', '.d64', '.t64'], undefined, 'C64', 'COMMODORE'),
+    // TODO(dcramer): multiple outputs for Batocera
+    new GameConsole(/Amiga/i, [], 'amiga', 'Amiga', 'AMIGA', undefined),
```

Comment:
MiSTer lumps a lot of these together (https://mister-devel.github.io/MkDocs_MiSTer/cores/computer/), this could probably get split out into multiple consoles to better support Batocera.

This can definitely happen in a different PR.